Libraries/src/Amazon.Lambda.AspNetCoreServer/HostBuilderExtensions.cs

```diff
@@ -42,6 +42,7 @@ public static IHostBuilder ConfigureWebHostLambdaDefaults(this IHostBuilder buil
                                 }
                                 else
                                 {
+                                    logging.ClearProviders();
```

Comment:
Without this both our logger and the console logger where added which is what triggered the behavior. Now we clear the console logger. Not to work around the deadlock, that is fixed by the locking fix. We clear because it fixes another reported issue about double logs in CloudWatch Logs.